dev_guide/ssh_environment.adoc

```diff
@@ -0,0 +1,37 @@
+= Managing Applications in a Secure Shell Environment
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Managing applications in a secure shell (SSH) environment provides specialized
```

Comment:
@tnguyen-rh, you're right and I keep beating my head off of the desk trying to figure out a better way to rewrite this. Thinking on it...